### PR TITLE
Feature: Info Page

### PR DESCRIPTION
**Pull request by NelsonJTSM**
https://github.com/KnightHacks/hackathon-app-2021/issues/9

### CODE REVIEW

**Review by suneettipirneni — CHANGES_REQUESTED** [review]
Looks good overall, just some minor changes.

File: src/components/cards/SocialCard.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { View, Text, Pressable, Linking } from 'react-native';
+
+export type Social = {
```
Comment (suneettipirneni) [inline src/components/cards/SocialCard.tsx:4]: use an `interface` instead of a `type` here.

File: src/components/cards/SocialCard.tsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  type: string;
```
Comment (suneettipirneni) [inline src/components/cards/SocialCard.tsx:5]: You can declare an `enum` representing the possible values for the `type` field:

```ts
enum SocialTypes {
  Discord,
  Website,
  Facebook,
  Instagram,
  Twitter,
}
```

Then set the `type` to be any of the keys from that enum:
```suggestion
  type: keyof SocialTypes;
```

File: src/components/cards/SocialCard.tsx [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  text: string;
+  url: string;
+  logo: any;
```
Comment (suneettipirneni) [inline src/components/cards/SocialCard.tsx:8]: I don't think logo should be `any` it should probably be `JSX.Element`

**Review by suneettipirneni — CHANGES_REQUESTED** [review]
Looks good overall, just some minor changes.

**Comment by NelsonJTSM** [comment]
Minor changes have been pushed.

**Review by suneettipirneni — APPROVED** [review]
Lgtm

**Pull request merged**